app/scripts/app/directives/materialize.select.js

```diff
@@ -26,7 +26,9 @@ angular.module("ngm.materialize.select", [])
 
 																// if searchable then prepend search
 																if (attrs.searchable) {
-																	addSearch();
+                                                                    setTimeout(() => {
```

Comment:
remove $('select').formSelect(); on details html, and no need for timeout here, it's better not to use page formSelect

Reply:
updated